src/adaptors/package.json

```diff
@@ -20,6 +20,9 @@
     "dotenv": "^10.0.0",
     "graphql": "^15.5.1",
     "graphql-request": "^3.5.0",
+    "hi-base32": "^0.5.1",
+    "js-sha512": "^0.8.0",
```

Comment:
are `hi-base32` and `js-sha512` actually used anywhere in your adapter? if not, pls remove

Reply:
Removed